utils/certs/gen_certificates.sh

```diff
@@ -38,146 +38,210 @@
 
 
 __usage="
-Usage: gen_certificates.sh [OPTIONS]
-Generate certificates for DAOS deployment in the ./daosCA.
+Usage: gen_certificates.sh [DIR]
+Generate certificates for DAOS deployment in the [DIR]/daosCA.
+By default [DIR] is the current directory.
 "
 
 function print_usage () {
-	>&2 echo "$__usage"
+    >&2 echo "$__usage"
 }
 
-PRIVATE="./daosCA/private"
-CERTS="./daosCA/certs"
+CA_HOME="${1:-.}/daosCA"
+PRIVATE="${CA_HOME}/private"
+CERTS="${CA_HOME}/certs"
 CONFIGS="$(dirname "$BASH_SOURCE")"
 
 function setup_directories () {
-	mkdir -p ./daosCA/{certs,private}
+    mkdir -p ${PRIVATE}
+    mkdir -p ${CERTS}
+}
+
+function generate_ca_cnf () {
+    echo "[ ca ]" > "${CA_HOME}/ca.cnf"
+    echo "default_ca = CA_daos" >> "${CA_HOME}/ca.cnf"
```

Comment:
(lint) Consider using { cmd1; cmd2; } >> file instead of individual redirects. [SC2129]